src/player.cpp

```diff
@@ -10483,11 +10483,19 @@ bool player::can_wield( const item &it, bool alert ) const
             if( alert ) {
                 add_msg( m_info, _("The %s can't be wielded with only one arm."), it.tname().c_str() );
             }
+            if( worn_with_flag("RESTRICT_HANDS") ) {
+                add_msg( m_info, _("Something you are wearing hinders the use of both hands."),
```

Comment:
How about this: 'The battleaxe you have equipped...' Works just as well for briefcase: 'The briefcase you have equipped...'
'you have equipped' could also be 'you are equipping'